web/.env.dev

```diff
@@ -1,3 +1,4 @@
 NODE_ENV=development
 __DEV__=true
-API_URL=https://depviz-demo.moul.io/api
+#API_URL=https://depviz-demo.moul.io/api
+API_URL=https://depviz-demo-auth.moul.io/api/
```

Comment:
This should not be hardcoded in the repo, feel free to create a new dedicated env file for testing authentication

but the default behavior should be to use unauthenticated depviz-demo